ui.frontend/src/view/FormPanel.js

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+import {Constants} from "../constants";
+import FormField from "./FormField";
+
+export default class FormPanel extends FormField {
```

Comment:
It can extend FormFieldBase

Reply:
What was the motivation of having two classes FormField and FieldBase?

Reply:
FormFIeld would be public for the customer since that can have any arbitrary HTML structure while FormFieldBase provides the base functionality for our components.

Reply:
ok, will extend from FieldBase.